PowerPointLabs/PowerPointLabs/PositionsLab/PositionsLabMain.cs

```diff
@@ -96,7 +96,8 @@ public static void AlignLeft(ShapeRange toAlign)
                 case PositionsLabSettings.AlignReferenceObject.SelectedShape:
                     if (toAlign.Count < 2)
                     {
-                        throw new Exception(PositionsLabText.ErrorFewerThanTwoSelection);
+                        // Insert default align left behaviour here
```

Comment:
There's reason to believe that the comments are quite useless, as we know that the `Align` method is the built-in PowerPoint align.

Reply:
@Skaty 

Agreed. It was for the purposes of identifying the point of adding the code while I was working on this. Ill remove them